porcupine/plugins/filemanager.py

```diff
@@ -371,6 +388,8 @@ def new_file_here(path: Path) -> None:
     Command(f"Move to {trash_name}", "<<FileManager:Trash>>", is_NOT_project_root, trash),
     Command("Delete", "<<FileManager:Delete>>", is_NOT_project_root, delete),
     Command("Open in file manager", None, (lambda p: p.is_dir()), open_in_file_manager),
+    Command("Open in terminal", None, (lambda p: p.is_dir()), open_in_terminal),
+    Command("Copy full path to clipboard", None, (lambda p: p.is_dir()), copy_full_path_to_clipboard),
```

Comment:
You should be able to copy any path whatsoever to clipboard, not just directories.
```suggestion
    Command("Copy full path to clipboard", None, (lambda p: True), copy_full_path_to_clipboard),
```